data-raw/R2R_TMH_habitat_inputs/documentation/inchannel_rearing_process_and_Review.Rmd

```diff
@@ -0,0 +1,208 @@
+---
+title: "In-channel Rearing Suitability for High Gradient Reaches"
+author: "Maddee Rubenson (FlowWest)"
+date: "`r Sys.Date()`"
+output:
+  word_document: default
+  html_document:
+    theme: paper
+    df_print: paged
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE,
+                      warning = FALSE, 
+                      message = FALSE)
+
+library(tidyverse)
+library(lubridate)
+
+```
+
+## Introduction
+
+This document provides details on the approach used to estimate an in-channel rearing suitable proportion value to be used in the theoretical maximum habitat process for the Reorienting to Recovery project. The approach uses a 2013 study by Stillwater Sciences, [*Modeling Habitat Capacity and Population Productivity for Spring-run Chinook Salmon and Steelhead in the Upper Yuba River Watershed*](http://www.ycwa-relicensing.com/Technical%20References/07%20-%20Threatened%20Endangered%20and%20Fully%20Protected%20Species/2012%20-%200221%20-%20Stillwater%20Sciences%20RIPPLE%20Report.pdf)[^1]*,* where the authors use the gradient of the North, South, and Middle Yuba River to calculate a proportion of the reach suitable for in-channel rearing at habitats (pool, riffle, and run) within the channel. The tables F-1, F-6, and 4-1 within the Stillwater Sciences report were used in this analysis.
+
+[^1]: Stillwater Sciences. 2012. Modeling habitat capacity and population productivity for spring-run Chinook salmon and steelhead in the Upper Yuba River watershed. Technical Report. Prepared by Stillwater Sciences, Berkeley, California for National Marine Fisheries Service, Santa Rosa, California.
+
+FlowWest used the process and data collected by Stillwater Sciences to develop an approach that can be used for all Reorienting to Recovery watersheds that are considered high gradient or valley foothills (outside of the [Habitat Quantification Tool](https://www.sfei.org/sites/default/files/biblio_files/SalmonHQT_UserGuide_2019.pdf) boundary and have a slope greater than 0.4%).
+
+## Objectives
+
+**Objective 1:** Is it possible to use a coarse estimate of gradient by dividing the reach length by the maximum and minimum elevation difference?
```

Comment:
This feels like a question to me not an objective - please relabel or rephrase

Reply:
reworded to: 

> Compare an rough gradient estimation method, which involves dividing the reach length by the difference between the maximum and minimum elevations, and a more detailed approach that subdivides the reach length into smaller segments.